mne/beamformer/_dics.py

```diff
@@ -140,14 +309,17 @@ def dics(evoked, forward, noise_csd, data_csd, reg=0.05, label=None,
     real_filter : bool
         If True, take only the real part of the cross-spectral-density matrices
         to compute real filters as in [2]_. Default is False.
+    picks : array-like of int
+        Channel indices to use for beamforming (if None all channels
+        are used except bad channels).
```

Comment:
I was  following ``lcmv`` (and ``make_lcmv``) here, which has a picking parameter. 

There could probably be a use case with the separate ``make_dics`` and ``apply_dics`` functions, when working with different data sets?